layers/+lang/csharp/funcs.el

```diff
@@ -0,0 +1,150 @@
+;;; funcs.el --- C# Layer functions File for Spacemacs
+;;
+;; Copyright (c) 2012-2019 Sylvain Benner & Contributors
+;;
+;; Author: Muneeb Shaikh <muneeb@reversehack.in>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+
+;; backend
+
+(defun spacemacs//csharp-backend ()
```

Comment:
This function is not necessary just directly reference `csharp-backend`, the point is lsp layer is always loaded when `csharp-backend` is `lsp` this makes this getter superfluous.

Reply:
I do believe it is useful, actually. Since csharp-backend is a nil'able variable and we want to default to *some* backend when resolving it our code.